### DISCUSSION TITLE
How to remove the link

### DISCUSSION THREAD

**Discussion by removeme001** [agorahub/blog0#15] [Q&A] [answered]
Agora published the following article without permission granted. How to remove it?

https://agora0.gitlab.io/pen/21q3/stand/2021/07/23/STAND-%E6%99%82%E4%BB%A3%E9%9D%A9%E5%91%BD-%E5%B0%8E%E6%BC%94%E5%91%A8%E5%86%A0%E5%A8%81-%E9%9B%BB%E5%BD%B1%E5%B0%B1%E6%98%AF%E5%89%B5%E9%80%A0%E5%8A%9B-%E6%A8%82%E8%A7%80%E6%98%AF%E5%8F%AF%E4%BB%A5%E5%89%B5%E9%80%A0%E7%9A%84.html

**Comment by theagora** [comment, marked as answer]
https://github.com/agorahub/blog0/discussions/14#discussioncomment-4597680
